test/notifier.test.js

```diff
@@ -1057,6 +1057,38 @@ describe("Notifier._buildPayload()", function() {
     done();
   });
 
+  it("should have the most recent frame last", function(done) {
+    var notifier = new Notifier();
+
+    function first() {
+      second();
+    }
+
+    function second() {
+      last();
+    }
+
+    function last() {
+      nonExistant(); 
+    }
+
+    var err;
+    try {
+      first();
+    } catch (e) {
+      err = e
+    }
+    var payload = notifier._buildPayload(new Date(), 'error', 'frame order test', TK(err));
+    expect(payload.data.body.trace.frames).to.be.an('array');
+
+    var numFrames = payload.data.body.trace.frames.length;
+    expect(payload.data.body.trace.frames[numFrames - 3].method).to.equal('first');
+    expect(payload.data.body.trace.frames[numFrames - 2].method).to.equal('second');
+    expect(payload.data.body.trace.frames[numFrames - 1].method).to.equal('last');
```

Comment:
Thanks. My brain must have been fried yesterday. Let's merge #82 and close this.